flink-autoscaler/src/main/java/org/apache/flink/autoscaler/metrics/ScalingMetric.java

```diff
@@ -39,12 +39,6 @@ public enum ScalingMetric {
     /** Current processing rate. */
     CURRENT_PROCESSING_RATE(true),
```

Comment:
Could all callers use the `getRate(ScalingMetric.NUM_RECORDS_IN, vertex, metricsHistory)` instead of `getAverage(CURRENT_PROCESSING_RATE, vertex, metricsHistory);`?

If yes, we can remove this metric.

Currently, only one caller use this metric.

Reply:
I would not remove that as it has a bit different semantics and is mainly used to determine whether the job is processing backlog or not

Reply:
On a second thought, you are right, I did not realise that the CURRENT_PROC_RATE was averaged, removing

Reply:
Thanks for the update.